lexos/templates/scrub.html

```diff
@@ -34,6 +34,7 @@
                 }
                                 });
            tagDict=JSON.stringify(tagDict);
+           console.log(tagDict);
```

Comment:
I don't know that it actually has a noticeable effect on performance, but it's technically debugging code and so should probably be commented out when not in use. And we should probably have a pre-release step of removing stray `console.log` statements, just as we do with `print` commands in Python.  But I think the time to adopt this practice is when we begin our first Javascript cleanup.